projects/cashmere-examples/src/lib/tabs-vertical/tabs-vertical-example.component.html

```diff
@@ -17,13 +17,22 @@
                         Horizontal Tab content 3
                     </div>
                 </hc-tab>
+                <div class="tab-bar-content">
+                    <div>
+                        <hc-icon fontSet="fa" fontIcon="fa-info"></hc-icon>
+                        <span>You can place additional content after the tabs</span>
+                    </div>
+                </div>
             </hc-tab-set>
         </hc-tab>
-        <hc-tab tabTitle="Tab 2">
+        <hc-tab tabTitle="Tooltip Tab" [hcPop]="tabTooltip" trigger="hover">
             <div class="tab-content">
-                Vertical Tab content 2
+                This tab shows a Cashmere popover tooltip when you hover over it.
```

Comment:
:joy:

Reply:
Can you provide a screenshot or two of the demo app?

Reply:
Ah right, good idea, one sec..

Reply:
![image](https://user-images.githubusercontent.com/5783377/65082326-a4270d80-d973-11e9-8655-eeb19b8def05.png)

![image](https://user-images.githubusercontent.com/5783377/65082337-aee1a280-d973-11e9-847b-bb794ff1002d.png)

Reply:
Yay thanks